<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>746</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_setting">
        <property name="text">
         <string>设置通达信数据路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_dayRise">
        <property name="text">
         <string>更新股票列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_select">
        <property name="text">
         <string>选股</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_pickup">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2016</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_useTDXdata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>使用通达信数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_macdCross">
        <property name="text">
         <string>MACD金叉</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_macdDivergence">
        <property name="text">
         <string>MACD底背离</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_savePickup">
        <property name="text">
         <string>保存选股结果</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_tdxDataPath"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>股票列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView_dayList">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>选股列表</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView_pickup">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_Setting"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="action_Setting">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_select</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Setting</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_setting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_setting</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_setting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_useTDXdata</sender>
   <signal>stateChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>do_use_tdx()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_about</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_dayRise</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_day_rise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>-5</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>do_stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>do_select()</slot>
  <slot>do_setting()</slot>
  <slot>do_use_tdx()</slot>
  <slot>do_macd_cross()</slot>
  <slot>do_macd_diverse()</slot>
  <slot>do_pickup_save()</slot>
  <slot>do_about()</slot>
  <slot>do_day_rise()</slot>
  <slot>do_stop()</slot>
 </slots>
</ui>
